src/pretalx/submission/models/submission.py

```diff
@@ -124,6 +124,14 @@ class Submission(LogMixin, models.Model):
     notes = models.TextField(
         null=True, blank=True, verbose_name=_('Notes for the organiser')
     )
+    internal_notes = models.TextField(
+        null=True,
+        blank=True,
+        verbose_name=_('Notes by the organiser'),
```

Comment:
I think calling the field 'Internal notes' would be better – otherwise you'll have "notes for the organiser" and "notes by the organiser" directly next to each other, and that would be more confusing than helpful.